test/Test/Hspec/GoldenSpec.hs

```diff
@@ -65,6 +65,11 @@ spec =
            goldenFileContent <- readFile goldenFilePath
            goldenFileContent `shouldBe` fixtureContent
 
+    context "when IO actions are needed in the test case" $
+      it "`defaultGolden` should be accessible from the test case" $ do
```

Comment:
Hey @dbalseiro Let's change the description to match the expectation as described in [better tests guide](https://bettertests.js.org/#/recommendations/05-dont-use-should). For example:
```suggestion
      it "enables `defaultGolden` to be accessible from the test case" $ do
```